lib/Models/GtfsCatalogItem.ts

```diff
@@ -258,7 +259,11 @@ export default class GtfsCatalogItem extends AsyncMappableMixin(
 
   @computed
   get mapItems(): DataSource[] {
-    return [this.dataSource];
+    if (isDefined(this.dataSource)) {
+      this._dataSource.show = this.show;
```

Comment:
The best place to set `_dataSource.show` might actually be inside the `get dataSource`. That's just my opinion :)

Reply:
That makes sense - will fix that

Reply:
Though I'm finding that when I set `_dataSource.show` in `get dataSource` it doesn't behave in the same way. Instead of hiding the images when the button is clicked, it only does that when the timer resets. Not sure why

Reply:
You are right, it seems to show/hide only when setting `_dataSource.show` inside of `mapItems`. Setting `_dataSources.show` inside `get dataSource` is not triggering the autorun `Cesium.ts` for some reason :thinking: 

I'll approve this for now.